gateway/test/services/ethereum-routes.test.ts

```diff
@@ -12,7 +13,7 @@ describe('Eth endpoints', () => {
     eth = Ethereum.getInstance();
```

Comment:
Since we're running multiple tests that involve monkey-patching methods in the Ethereum object - it'd be cleaner if we initialize a new Ethereum object for each test, rather than sharing the same object. This would prevent previous methods or states in one test case from leaking into a latter test case.

Reply:
Actually... let's just drop this comment for the moment. Looks like we need a more general solution to the mock leakage issue here. I'm gonna file a separate task to find a good solution for all test cases.